webpack.config.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
   output: {
     chunkFilename: '[name].js',
     path: path.join(__dirname, '/build'),
-    filename: 'bundle.js',
+    filename: '[name].js',
```

Comment:
Just a little tweak to our bundle names. Now it will default to `main`. For other entry chunks it will use the object property key (i.e. `{ app: 'src/app.js' }`)